front/app/components/ConsentManager/Banner.tsx

```diff
@@ -104,19 +104,19 @@ const ButtonContainer = styled.div`
 `;
 
 const PreferencesButton = styled(Button)`
-  margin-right: 10px;
+  margin-left: 10px;
 
   ${media.smallerThanMinTablet`
     margin-right: 0px;
     order: 2;
   `}
 
   ${isRtl`
-    margin-right: 0;
-    margin-left: 10px;
+    margin-right: 10px;
+    margin-left: 0px;
 
     ${media.smallerThanMinTablet`
-        margin-left: 0px;
+        margin-right: 0px;
     `}
   `}
```

Comment:
Generally, for non-RTL, we try to keep margin on the right and bottom, not top/left, to avoid margin clashes in the future. It's not perfect throughout the code base, but here, I would instead add the margin-right to AcceptButton. (For RTL, I think we want to do the opposite. Also bottom > top, but left instead of right by default.

Reply:
I will make this change.

Reply:
@brentguf I have changed to using `margin-right` on the Accept button.

I'm not sure if you are advising any other changes when you say "Also bottom over top margin, but left instead of right by default." Could you clarify if more changes are required / desired? Thanks.

Reply:
Will have a look, please re-request me when a new review is needed! That way I can keep an overview in https://github.com/pulls/review-requested

Reply:
I wasn't sure new review is needed. Depended on the answer to my Q. :-)

Reply:
Always good to re-review after changes!